src/app/public/modules/wait/wait-adapter.service.ts

```diff
@@ -21,13 +36,95 @@ export class SkyWaitAdapterService {
     waitEl: ElementRef,
     isFullPage: boolean,
     isWaiting: boolean,
-    isNonBlocking = false
-  ): void {
+    id: string,
+    waitCmp: SkyWaitComponent
```

Comment:
Style preference question: Now that we're passing the `SkyWaitComponent` as one of these arguments, why pass all the others when we can derive those from `waitCmp` (with the exception of `isWaiting`)?

Reply:
Oh I had forgotten that I had added id here. That one can come out and isNonBlocking I guess will have to be put back in. Unfortunately I have to maintain the same params it had before, because it is a public method 😞 
I'll also need to default waitCmp to undefined. Nice catch